src/io/flutter/actions/InstallSdkAction.java

```diff
@@ -17,6 +34,11 @@
     abstract void perform();
 
     abstract String getLinkText();
+
+    void showError(@NotNull String title, @NotNull String message) {
+      //TODO(pq): implement
```

Comment:
We do have some error classes we can hook this up to (we'd want to make sure that they don't fall over when called from a model dialog - from the wizard).